components/sales/SalesTable.vue

```diff
@@ -0,0 +1,189 @@
+              {{ props.row.relDate }}
+            </a>
+          </b-tooltip>
+        </div>
+      </b-table-column>
+
+      <b-table-column
+        v-slot="props"
+        cell-class="is-vcentered"
+        field="salePrice"
+        label="Price of sale">
+        <Money :value="props.row.salePrice" inline />
+      </b-table-column>
+
+      <template #empty>
+        <div v-if="!$fetchState.pending" class="has-text-centered">
+          {{ $t('spotlight.empty') }}
+        </div>
+        <b-skeleton :active="$fetchState.pending" />
+      </template>
+    </b-table>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, mixins } from 'nuxt-property-decorator'
+import { RowSeries } from './types'
+import salesFeedGql from '@/queries/rmrk/subsquid/salesFeed.graphql'
+import { sanitizeIpfsUrl } from '@/components/rmrk/utils'
+import formatDistanceToNow from 'date-fns/formatDistanceToNow'
+import { urlBuilderBlockNumber } from '@/utils/explorerGuide'
+
+import PrefixMixin from '~/utils/mixins/prefixMixin'
+
+const components = {
+  Identity: () => import('@/components/shared/format/Identity.vue'),
+  Money: () => import('@/components/shared/format/Money.vue'),
+  Loader: () => import('@/components/shared/Loader.vue'),
+  BasicImage: () => import('@/components/shared/view/BasicImage.vue'),
+  BasicPopup: () => import('@/components/shared/view/BasicPopup.vue'),
+}
+
+@Component({ components })
+export default class SalesTable extends mixins(PrefixMixin) {
+  protected data: RowSeries[] = []
+
+  async fetch() {
+    await this.fetchSalesFeed()
+  }
+
+  protected paseDate(ts: number): string {
+    return new Date(ts).toLocaleString('en-GB', {
+      timeZone: 'UTC',
+      timeZoneName: 'short',
+    })
+  }
+
+  protected getBlockUrl(block: string): string {
+    return urlBuilderBlockNumber(
+      block,
+      this.$store.getters['explorer/getCurrentChain'],
+      'subscan'
+    )
+  }
+
+  public async fetchSalesFeed() {
+    const {
+      data: { salesFeed },
+    } = await this.$apollo.query({
+      query: salesFeedGql,
+      client: this.client,
+      variables: {},
+    })
+
+    salesFeed.forEach((nft, idx) => {
+      nft.idx = idx + 1
+      nft.image = sanitizeIpfsUrl(nft.image)
+      nft.date = this.paseDate(Number(nft.timestamp))
+      nft.relDate = formatDistanceToNow(Number(nft.timestamp), {
+        addSuffix: true,
+      })
+    })
+
+    this.data = salesFeed
+  }
+}
+</script>
+<style lang="scss">
+@import '@/styles/variables';
+
+/* ??? global */
+.b-radio.is-selected {
+  color: #000;
+  background-color: $primary;
+}
+
+.history {
+  width: 200px;
+  height: 100px;
+}
+
+.series-sticky-header th {
+  top: 120px;
+  position: sticky;
+  background: $frosted-glass-background;
+  backdrop-filter: $frosted-glass-backdrop-filter;
+}
+
+.front-stack-layer {
+  z-index: 1;
+}
+.popup-image {
+  width: 300px;
+  height: 300px;
+}
+</style>
```

Comment:
I know that its probably done the same in SeriesTable but we should really worry about polluting global namespace with all this stuff.

Reply:
maybe put all the global stuff from SeriesTabel, Salestable, etc into own scss file. but can be done in followup task

Reply:
hm, maybe scope first, and then I would extract that SCSS into one file for those stats pages.

I would prefer no global stuff since it would unconsciously break something.

Reply:
FYI, For now, I just converted it as scoped. After some PR of stats merged, I would start refactoring.